### PR TITLE
Add skip rmsnorm[SiliconFlow]

### PR DESCRIPTION
**Pull request by MARD1NO**

### CODE REVIEW

File: src/flag_gems/__init__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def enable(lib=aten_lib):
     lib.impl("isinf", isinf, "CUDA")
     lib.impl("isnan", isnan, "CUDA")
     lib.impl("native_layer_norm", layer_norm, "AutogradCUDA")
+    lib.impl("skip_rms_norm", skip_rms_norm, "CUDA")
```
Comment (StrongSpoon) [inline src/flag_gems/__init__.py]: same as skip_layernorm. delete this line.

Comment (MARD1NO) [reply src/flag_gems/__init__.py]: Done

File: src/flag_gems/fused/__init__.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 from .skip_layernorm import skip_layer_norm
+from .skip_rms_norm import skip_rms_norm
 from .silu_and_mul import silu_and_mul
 from .gelu_and_mul import gelu_and_mul
 
 
 __all__ = [
+    "skip_layer_norm", 
+    "skip_rms_norm",
```
Comment (StrongSpoon) [inline src/flag_gems/fused/__init__.py:9]: delete one "skip_layer_norm"

File: src/flag_gems/fused/skip_rms_norm.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    mask = tl.arange(0, BLOCK_SIZE) < N
+    cols = tl.arange(0, BLOCK_SIZE)
+    x = tl.load(X + cols * x_stride_c, mask, other=0.0).to(tl.float32)
+    r = tl.load(R + cols * r_stride_c, mask, other=0.0).to(tl.float32)
+
+    x += r 
+
+    var = tl.sum(x * x / N, axis=0)
+    rrms = 1 / tl.sqrt(var + eps)
```
Comment (tongxin) [inline src/flag_gems/fused/skip_rms_norm.py:40]: How about using tl.rsqrt instead?

**Pull request merged**